BorrowBooks/features/home/home.js

```diff
@@ -0,0 +1,46 @@
+document.addEventListener("DOMContentLoaded", function() {
+    const bookItems = document.querySelectorAll(".book-item");
+    const bookDetailCard = document.getElementById("bookDetailCard");
+    const closeBtn = document.getElementById("closeBtn");
+    const homePage = document.querySelector(".home-page");
+
+    bookItems.forEach(item => {
+        item.addEventListener("click", function() {
+            const image = this.getAttribute("data-image");
+            const title = this.getAttribute("data-title");
+            const author = this.getAttribute("data-author");
+            const description = this.getAttribute("data-description");
+
+            console.log('Title:', title);
+            console.log('Author:', author);
+            console.log('Description:', description);
+            console.log('Image:', document.getElementById("bookImage").scr);
```

Comment:
This should be removed.

Reply:
You are right. It was used for testing. The change is applied now.